source/backend/routes/recipes.js

```diff
@@ -1,10 +1,32 @@
 const express = require('express');
+const recipesModel = require('../database/models/recipesModel');
 
 const router = express.Router();
 
 /* GET /recipes */
-router.get('/', (req, res) => {
-  res.send('recipes');
+router.get('/:id', async (req, res) => {
+  res.send('chicken');
+  try {
+    const { recipeId } = req.params.getByRecipeId;
+    //if (parseInt(req.params.id, 10) !== recipeId) {
```

Comment:
We can probably remove the comments here too

Reply:
Awesome will do